searchlib/src/vespa/searchlib/queryeval/nearest_neighbor_iterator.h

```diff
@@ -25,17 +25,20 @@ class NearestNeighborIterator : public SearchIterator
         const DenseTensorView &queryTensor;
         const DenseTensorAttribute &tensorAttribute;
         NearestNeighborDistanceHeap &distanceHeap;
+        const search::BitVector *bit_vector;
```

Comment:
Consider renaming _bit_vector_ to _filter_ to match how it is named in hnsw index. Applies for rest of file.